models/yang/sonic/sonic-tam.yang

```diff
@@ -0,0 +1,69 @@
+module sonic-tam {
+    namespace "http://github.com/Azure/sonic-tam";
+    prefix tam;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC TAM";
+
+    revision 2019-11-08 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-tam {
+
+        container TAM_DEVICE_TABLE {
+
+            list TAM_DEVICE_TABLE_LIST {
+                key "device";
+                max-elements "1";
+
+                leaf device {
+                    type string;
+                    default "device";
```

Comment:
Can the key have any other string apart from "device" ? If not define as enum instead of string or have pattern for string type.

Reply:
Done